src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/ResourcesSpec.scala

```diff
@@ -91,8 +98,10 @@ class ResourcesSpec extends BaseSpec with Eventually with Inspectors with Cancel
       val resolverPayload =
         jsonContentOf("/kg/resources/cross-project-resolver.json", Map(quote("{project}") -> id1))
 
-      cl(Req(POST, s"$kgBase/resolvers/$id2/", headersUser, resolverPayload.toEntity)).mapResp { result =>
-        result.status shouldEqual StatusCodes.Created
+      eventually {
```

Comment:
`eventually` around a `POST` request seems weird, why do we need it?

Reply:
We need it, because we have no guarantee that the project has already been indexed in KG by the time we're making this request.